docs/proposals/compute-resource-metrics-api.md

```diff
@@ -35,15 +35,31 @@ Documentation for other releases can be found at
 
 ## Goals
 
-Provide resource usage metrics on pods and nodes on the API server to be used
-by the scheduler to improve job placement, utilization, etc. and by end users
-to understand the resource utilization of their jobs. Horizontal and vertical
-auto-scaling are also near-term uses.
+Provide resource usage metrics on pods and nodes through the API server to be
+used by the scheduler to improve job placement, utilization, etc. and by end
+users to understand the resource utilization of their jobs. Horizontal and
+vertical auto-scaling are also near-term uses. Additionally, a subset of the
```

Comment:
In the future, I can see kubelet might exposing `derived` metrics, which will help us scale heapster as well. I don't see why we will have two types if the versions match in that case.

Perhaps, we can state the general APIs in the beginning, and then explicitly state which component will be exposing these APIs?